book/chapters/chapter12/model_interpretation.qmd

```diff
@@ -296,7 +335,9 @@ For our `tsk("german_credit")` example, we might consider what changes in featur
 knitr::include_graphics("Figures/counterfactuals.png", dpi = 600)
 ```
 
-A simple counterfactual method is the `r index('What-If', aside = TRUE, lower = FALSE)` approach (@Wexler2019) where, for a given prediction to explain, the counterfactual is the closest data point in the dataset with the desired prediction -- so if the underlying dataset is changed then the counterfactual can be quite different.
+A simple counterfactual method is the `r index('What-If', aside = TRUE, lower = FALSE)` approach (@Wexler2019) where, for a given prediction to explain, the counterfactual is the closest data point in the dataset with the desired prediction.
+Obviously, if we do not restrict ourselves to a fixed dataset, many possible counterfactual data points can exist, as there are many ways to achieve a desired prediction.
+However, the approach by @Wexler2019 only produces a single counterfactual explanation, as many other early counterfactual methods (see @guidotti2022counterfactual for a comprehensive overview).
```

Comment:
@mllg @giuseppec I think there's a typo here, I can't follow this sentence

Reply:
rewrite like this:
A simple counterfactual method is the `r index('What-If', aside = TRUE, lower = FALSE)` approach (@Wexler2019) where, for a given prediction to explain, the counterfactual is the closest data point in the dataset with the desired prediction.
Usually, many possible counterfactual data points can exist. However, the approach by @Wexler2019, and many other early counterfactual methods for that matter (see @guidotti2022counterfactual for a comprehensive overview), only produce a single, somewhat arbitrary counterfactual explanation, which can be regarded as problematic when counterfactuals are used for insights or actions against the model.